src/plugins/es5.ts

```diff
@@ -7,9 +7,8 @@ export const es5 = (): Plugin => {
   return {
     name: 'es5-target',
 
-    esbuildOptions(options) {
-      if (options.target === 'es5') {
-        options.target = 'es2020'
```

Comment:
This still worries me. I think I will get an error from esbuild about it not supporting es5, rather than esbuild working with es2020 and then swc transpiling it back to es5?

Reply:
I know what you worry about. See here, the usage:
- --target=es5 and tsconfig.compilerOptions.target: "es5" can not use at the same time.
- can only use --target es5 to compile the code down to es5.

@ComLock

Reply:
It was a pleasure to work with you. @ComLock

Reply:
Neither target on cmdline, in tsup.config.json nor tsconfig.custom.json works. All gives es5 not supported by esbuild errors.

Reply:
That's just ok.

Reply:
In order to avoid the error I have to set esbuildOptions target: 'es2020' in addition to 'es5' (cmdline or tsup.config or tsconfig.custom)

Reply:
can you push the change? @ComLock